website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

```diff
@@ -57,9 +57,10 @@ The IDE features some delightful tools and layouts to make it easier for you to
 
 2. **Save button &mdash;** The editor has a **Save** button that saves editable files. Pressing the button or using the Command-S or Control-S shortcut saves the file contents. You don't need to save to preview code results in the Console section, but it's necessary before changes appear in a dbt invocation. The File Editor tab shows a blue icon for unsaved changes.
 
-3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, or creating a merge/pull request when appropriate. The dropdown menu on the Git actions button allows users to revert changes, refresh git state, create merge/pull requests, and change branches. You can also [resolve merge conflicts](/docs/collaborate/git/merge-conflicts) and for more info on git, refer to [Version control basics](/docs/collaborate/git/version-control-basics#the-git-button-in-the-cloud-ide).
-
-   - **Version Control Options menu &mdash;** Below the Git Actions button, the **Changes** section, which lists all file changes since the last commit. You can click on a change to open the Git Diff View to see the inline changes. You can also right-click any file and use the file-specific options in the Version Control Options menu.
+3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, or creating a merge/pull request when appropriate.
+   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, and change branches. Keep in mind, you can't delete branches from the remote branches directly using this menu. However, if you [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus:~:text=change%20their%20branch.-,IDE%20Options%20menu,-%E2%80%94%20The%20IDE), it will hide deleted branches from the **Change Branch** list.
```

Comment:
```suggestion
   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, and change branches. 
      - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus:~:text=change%20their%20branch.-,IDE%20Options%20menu,-%E2%80%94%20The%20IDE). Doing this will hide deleted branches from the **Change Branch** list.
```